controllers/kserve_inferenceservice_controller_test.go

```diff
@@ -19,56 +19,66 @@ import (
 	"context"
 	kservev1alpha1 "github.com/kserve/kserve/pkg/apis/serving/v1alpha1"
```

Comment:
I would recommend to separate imports for better readability as a standard best practice.
Also sorting affects package initialization, first third party then local ones etc.